public/js/webcam.js

```diff
@@ -26,10 +29,16 @@ video.addEventListener("animationend", ()=> {
 async function init() {
     try {
         hide([canvas, afterCaptureButtons, beforeCaptureButtons, loadingDiv]);
+        hide([errorAlert]);
         stream = await navigator.mediaDevices.getUserMedia(constraints);
         handleSuccess(stream);
     } catch (err) {
-        errorMsgElement.innerHTML = `navigator.getUserMedia.error:${err.toString()}`;
+        show([errorAlert]);
+        $("#errorAlert").append("<span>"+"Sorry, we encountered an error. Redirecting you in 5 seconds ..." +
+            error.response.data.errorMsg +"</span>");
+        setTimeout(function() {
+            location.href = "/";
+        }, 5000);
```

Comment:
We aren't doing anymore redirecting right?

Reply:
YES! My bad!